guide/README.md

```diff
@@ -23,5 +23,9 @@ If you don't know JS but would like to learn about it, here are a few links to h
 * [Nodeschool, for both JavaScript and Node.js lessons](https://nodeschool.io/)
 * [MDN's JavaScript guide and full documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
 * [Google, your best friend](https://google.com)
+* [Grasshopper, a JavaScript learning app by Google](https://grasshopper.app)
+* [w3schools' free tutorials for both server-side and client-side JavaScript](https://w3schools.com)
```

Comment:
:x: w3schools is unfortunately pretty badly structured, lacks information and reference wise [MDN ](https://developer.mozilla.org/) is superior in almost every aspect